README.md

```diff
@@ -83,3 +83,11 @@ datasets: [{
 In order to provide an alternative view for visually impaired viewers it's a good idea to [provide a patterned alternative](http://betweentwobrackets.com/data-graphics-and-colour-vision/#patternstotherescue).
 
 See the [pattern option example page](examples/optional.html) for a simple implementation of a pattern toggle switch for Chart.js.
+
+##Typescript / Angular
```

Comment:
@miller45 If you can just clean up the formatting of the Markdown here I should be able to get this merged. You can check it easily in the GitHub editor.

Reply:
@ashiguruma ok I added a space and pushed resolve